extension.json

```diff
@@ -47,7 +47,7 @@
 		},
 		{
 			"path": "/wikibase-rdf/v0/mappings/{entity_id}",
-			"method": [ "PUT" ],
+			"method": [ "POST" ],
```

Comment:
I changed the API to POST because the MW REST API client doesn't have PUT in MW 1.37 it seems. No idea why, but being strict on the REST verbs isn't too important right now.